Microsoft.Identity.Web/ClaimPrincipalExtension.cs

```diff
@@ -1,65 +1,12 @@
-﻿/************************************************************************************************
-The MIT License (MIT)
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
 
-Copyright (c) 2015 Microsoft Corporation
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
-***********************************************************************************************/
-
-using Microsoft.Identity.Client;
 using System.Security.Claims;
 
 namespace Microsoft.Identity.Web
 {
     public static class ClaimsPrincipalExtension
     {
-        /// <summary>
```

Comment:
Reusable code, not used here, but used in other samples ... 
Do we want to reconsisder having the whole library (which will be released as a nuget package anyway, so that's a point in time question)

Reply:
i think we should maybe discuss some things on this.

---
In reply to: [318912439](https://github.com/Azure-Samples/active-directory-dotnet-native-aspnetcore-v2/pull/93#discussion_r318912439) [](ancestors = 318912439)

Reply:
sure